### PR TITLE
Document child entity filtering

### PR DESCRIPTION
**Pull request by kamilkisiela**

### CODE REVIEW

File: pages/en/developer/graphql-api.mdx [hunk 1]
```diff
@@ -176,7 +195,7 @@ _not_ends_with
 _not_ends_with_nocase
 ```
 
-> Please note that some suffixes are only supported for specific types. For example, `Boolean` only supports `_not`, `_in`, and `_not_in`.
+> Please note that some suffixes are only supported for specific types. For example, `Boolean` only supports `_not`, `_in`, and `_not_in`, but `_` is available only for object and interface types.
```
[the remark this answers is not in the document]
Comment (kamilkisiela) [reply pages/en/developer/graphql-api.mdx:198]: I described here only the output types of GraphQL, both `NonNullable` and `List` are just modifiers. Maybe it's misleading? On the other hand, if we mention here the List modifier, then the NonNullable modifier is equal from the GraphQL standpoint and worth including as well, but then it becomes too complex.

Not sure, your choice :)

File: pages/en/developer/graphql-api.mdx [hunk 2]
```diff
@@ -150,11 +150,30 @@ This can be useful if you are looking to fetch only entities which have changed,
 }
 ```
 
+#### Example for child entity filtering
```
[the remark this answers is not in the document]
Comment (kamilkisiela) [reply pages/en/developer/graphql-api.mdx:153]: Yes, nested makes more sense to me as well.

**Comment by kamilkisiela** [comment]
@azf20 I think we can merge it now, no?

**Review by azf20 — APPROVED** [review]
LGTM!

**Pull request merged**